apps/form-management-app/src/app/pages/admin/FormEditor/Editor.scss

```diff
@@ -20,19 +20,35 @@
   font-weight: var(--fw-regular);
 }
 
+.PRE {
+  background: #f3f3f3;
```

Comment:
Hi @jonathanweyermann, react CSS and react CSS modules are different from compiling point of view.

Reply:
which one should I be using ?

Reply:
Hi @jonathanweyermann, my understanding is that we shall use the CSS module in this repo.